init/deviceref.go

```diff
@@ -3,6 +3,8 @@ package main
 import (
 	"bytes"
 	"fmt"
+	"github.com/foxboron/go-uefi/efi/attributes"
```

Comment:
third-party imports should be at the bottom of this section divided by an empty line